docs/en/stack/ml/df-analytics/flightdata-regression.asciidoc

```diff
@@ -0,0 +1,367 @@
+index time.
+
+. Create a {dfanalytics-job}.
++
+--
+Use the {ref}/put-dfanalytics.html[create {dfanalytics-jobs}] API as you can see 
+in the following example:
+
+[source,console]
+--------------------------------------------------
+PUT _ml/data_frame/analytics/model-flight-delays
+{
+  "source": {
+    "index": [
+      "kibana_sample_data_flights" <1>
+    ],
+    "query": { <2>
+      "range": {
+        "DistanceKilometers": { 
+          "gt": 0
+        }
+      }
+    }
+  },
+  "dest": {
+    "index": "df-flight-delays",  <3>
+    "results_field": "ml" 
+  },
+  "analysis": {
+    "regression": {
+      "dependent_variable": "FlightDelayMin",  <4>
+      "training_percent": 90  <5>
+    }
+  },
+  "analyzed_fields": {
+    "includes": [],
+    "excludes": [    <6>
+      "Cancelled",
+      "FlightDelay",
+      "FlightDelayType"
+    ]
+  },
+  "model_memory_limit": "100mb" <7>
+}
+--------------------------------------------------
+// TEST[skip:setup kibana sample data]
+
+<1> The source index to analyze.
+<2> This query removes erroneous data from the analysis to improve its quality.
+<3> The index that contains the results of the analysis; it consists of a copy 
+of the source index data where each document is annotated with the results.
+<4> Specifies the continuous variable we want to predict with the {reganalysis}.
+<5> Specifies the proportion of data that is used for training the model. In 
+this example we use 90% of the source data. The training data is randomly 
+selected and provide predictions from the remaining 10%. These percentages are 
+approximate.
+<6> Specifies fields to be excluded from the analysis. It is recommended to 
+exclude fields that either contain erroneous data or describe the 
+`dependent_variable`.
+<7> Specifies a memory limit for the job. If the job requires more than this 
+amount of memory, it fails to start. This makes it possible to prevent job 
+execution if the available memory on the node is limited.
+
+
+The API returns the following response:
+
+[source,console-result]
+--------------------------------------------------  
+{
+  "id" : "model-flight-delays",
+  "source" : {
+    "index" : [
+      "kibana_sample_data_flights"
+    ],
+    "query" : {
+      "range" : {
+        "DistanceKilometers" : {
+          "gt" : 0
+        }
+      }
+    }
+  },
+  "dest" : {
+    "index" : "df-flight-delays",
+    "results_field" : "ml"
+  },
+  "analysis" : {
+    "regression" : {
+      "dependent_variable" : "FlightDelayMin",
+      "training_percent" : 90.0
+    }
+  },
+  "analyzed_fields" : {
+    "includes" : [ ],
+    "excludes" : [
+      "Cancelled",
+      "FlightDelay",
+      "FlightDelayType"
+    ]
+  },
+  "model_memory_limit" : "100mb",
+  "create_time" : 1571918329132,
+  "version" : "7.5.0",
+  "allow_lazy_start" : false
+}
+--------------------------------------------------
+--
+
+. Start the job.
++
+--
+Use the {ref}/start-dfanalytics.html[start {dfanalytics-jobs}] and 
```

Comment:
If we mention `_stop` here, we should also mention earlier that they will not need to stop this analysis as it will stop itself once the analysis is complete.